docs/GettingStarted.md

```diff
@@ -28,7 +28,8 @@ The following must be installed in order to build this project:
    - `"MSVC v143 - VS 2022 C++ x64/x86 Spectre-mitigated libs (latest)"`
 
 1. [Visual Studio Build Tools 2022](https://aka.ms/vs/17/release/vs_buildtools.exe) (version **17.4.2 or later**).
-1. The *WiX Toolset* has a dependency on the **.NET 3.5 Framework**: you can either enable from the Start menu -> "*Turn Windows features on or off*" and then select "*.NET Framework 3.5 (includes .NET 2.0 and 3.0)*" (recommended), *or*
+1. [The WiX Toolset v3.11.2 build tools](https://github.com/wixtoolset/wix3/releases)
+    > Note: The *WiX Toolset* has a dependency on the **.NET 3.5 Framework**: you can either enable from the Start menu -> "*Turn Windows features on or off*" and then select "*.NET Framework 3.5 (includes .NET 2.0 and 3.0)*" (recommended), *or*
```

Comment:
This is fine for this release, but in the future, can the MSI automatically install dependencies that aren't installed yet?

Reply:
The Wix SDK is required for building Wix projects (i.e. candle.exe & light.exe).